recipe/meta.yaml

```diff
@@ -12,24 +12,24 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<35]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<37 or s390x]
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
```

Comment:
```suggestion
```

Because flit is being used, setuptools and wheels aren't required.

Reply:
Since the command ran is `pip install`, the build backend (flit-core in this case) creates the wheel and then pip installs it. So I'd be extremely surprised if conda-build would extract the wheel and redo it. Or in fact even modify it.

Reply:
And for reference, flit creates wheels without using the wheel package, see https://github.com/pypa/flit/blob/main/flit_core/flit_core/wheel.py.

Reply:
Ah, you are right, `flit-core` writes a simple `zip` file with `whl` file extension https://github.com/pypa/flit/blob/c8ae08dc9f3f067feeec9dfd2c443db592f4d3d1/flit_core/flit_core/wheel.py#LL77C12-L77C12

Reply:
Well, wheels are zip files with a standardized file structure :)

Reply:
Ahh I see. It seems to be a standard to add setuptools and wheel to a Python package, so this is good to know :) I'll remove the two. Thanks guys.